src/document/json/object-reference-html.xsl

```diff
@@ -65,7 +65,7 @@
       <div class="model-entry definition { tokenize($me/@_metaschema-json-id,'/')[2] }">
          <xsl:variable name="header-class" expand-text="true">{ if (exists(parent::map)) then
             'definition' else 'instance' }-header</xsl:variable>
-         <div class="{ $header-class }">
+         <div class="{ $header-class }{ if (exists($me/@deprecated)) then ' deprecated-header' else '' }">
```

Comment:
In addition to tightening the path I might put this expression into the `$header-class` variable, as it already templates this ... no biggie, certainly.